### DISCUSSION TITLE
List of listings

### DISCUSSION THREAD

**Discussion by ikeronnyk** [joaomlourenco/novathesis#332] [Q&A] [answered]
May I please know how I can get rid of the list of listings from the thesis report? Thank you.
<img width="1440" alt="List of listings" src="https://user-images.githubusercontent.com/24888452/168792003-a8b9f343-3577-4ce6-a564-3ee20e3dc2e5.png">

<img width="1440" alt="List of listings" src="https://user-images.githubusercontent.com/24888452/168791096-b41fbcb5-3ae0-4a20-a75b-a63a3d30e54b.png">

**Comment by joaomlourenco** [comment, marked as answer]
Just open "5_packages.tex" and comment or remove the
```latex
\usepackage{listings}
```

**Reply by FranciscoCaldas** [reply]
% \usepackage{listings}  is commented but i still can't remove it

**Reply by joaomlourenco** [reply]
Either switch to the latest version (v8.0.3) that has an option to skip "List of <STUFF>" or comment the proper line in the file `0-Config/6_list_of.tex`

```bash
\ntaddlistof{listoflistings}     % The List of Listings. Comment to omit.
```

**Comment by ikeronnyk** [comment]
Dear Prof, thank you for the response. It worked.
